presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotBrokerPageSourceBase.java

```diff
@@ -0,0 +1,384 @@
+/*
```

Comment:
I am wondering if this file is same as PinotBrokerPageSource.java ? I kind of skipped reviewing it in detail :-)

Reply:
mostly, just abstract it :)